README.de.md

```diff
@@ -0,0 +1,104 @@
+- [cyclelayouts](https://dwm.suckless.org/patches/cyclelayouts/dwm-cyclelayouts-20180524-6.2.diff)
+
+Um dwm auf deinem system installieren musst du diese Abhängigkeiten installieren:
+```bash
+yay -S nerd-fonts-ubuntu-mono
+```
+Ich nutze immer die Schriftart und icons.
+Du baurachst auch meine eigene Version von 
+**[dwmblocks](https://github.com/antoniosarosi/dwm/tree/master/dwmblocks)**
+und die **[~/.local/bin](https://github.com/antoniosarosi/dotfiles/tree/master/.local/bin)**
+Scripts.
+
+```bash
+# dwm & dwmblocks
+cd ~/.config
+git clone https://github.com/antoniosarosi/dwm.git
+mkdir -p ~/.local/share/dwm
+ln -s ~/.config/dwm/autostart.sh ~/.local/share/dwm
+
+# Scripts
+mkdir -p ~/.local/bin
+cd ~/.local/bin
+curl -sL "https://raw.githubusercontent.com/antoniosarosi/dotfiles/master/.local/bin/battery" -o battery
+curl -sL "https://raw.githubusercontent.com/antoniosarosi/dotfiles/master/.local/bin/volume" -o volume
+curl -sL "https://raw.githubusercontent.com/antoniosarosi/dotfiles/master/.local/bin/percentage" -o percentage
+curl -sL "https://raw.githubusercontent.com/antoniosarosi/dotfiles/master/.local/bin/brightness" -o brightness
+chmod 755 battery volume percentage brightness
+
+# Diese scripts haben
+sudo pacman -S pacman-contrib brightnessctl pamixer upower
+```
+
+Schreibe das in deine **~/.xprofile** Datei:
+
+```bash
+export PATH=$HOME/.local/bin:$PATH
+```
+
+Erstelle *dwm* und *dwmblocks* und erstelle eine neue *xsession*:
+
+```bash
+cd ~/.config/dwm
+sudo make clean install
+cd ~/.config/dwm/dwmblocks
+sudo make clean install
+sudo cp ~/.config/dwm/dwm.desktop /usr/share/xsessions
+```
+
+Die *autostart* Datei befindet sich in **~/.dwm/autostart.sh**.
+Teste es mitTest it with **[Xephyr](https://wiki.archlinux.org/index.php/Xephyr)**:
+
+```bash
+Xephyr -br -ac -noreset -screen 1280x720 :1 &
+DISPLAY=:1 dwm
+```
+
+If you want to modify bar icons, open **~/.config/dwmblocks/config.h**
+and change these lines:
+
+```cpp
+static const Block blocks[] = {
+//   Icon    Command                          Update Interval     Update Signal
+    { "  ", "checkupdates | wc -l",                 60,               0 },
+    { "",    "brightness",                           2,                0 },
+    { "",    "volume",                               2,                0 },
+    { "",    "battery",                              60,               0 },
+    { "",    "date '+ %d/%m/%Y   %H:%M%p'",        5,                0 },
+};
+```
+
+Then recompile *dwmblocks* and restart *dwm* using **mod + control + r**.
+
+```bash
+cd ~/.config/dwm/dwmblocks
+sudo make clean install
+```
+
+Once that's done, you can login. But keep in mind keybindings will not work
+unless you have the same programs that I use and the same configs. You can
+either change keybindings or install the software I use and my config files,
+check out [my dotfiles repo](https://github.com/antoniosarosi/dotfiles#keybindings)
+for instructions.
```

Comment:
Not translated

Reply:
oh thats a fale version i am sorry

Reply:
Push the final version :)

Reply:
i have some issues with emacs i could take some time maybe 5 minutes my new version was destroyed by emas

Reply:
No problem, don't rush it, take your time.